cylc/flow/network/schema.py

```diff
@@ -1918,6 +1947,8 @@ class Meta:
             tasks will submit immediately if triggered, even if that violates
             the queue limit (so you may need to trigger a queue-limited task
             twice to get it to submit immediately).
+
+            Valid for: running workflows.
```

Comment:
Same as above, technically valid, but nothing will happen until the workflow is resumed which could cause confusion. However, there are probably valid use cases for pausing a workflow, mucking about with the graph, then resuming it so probs best to permit it.

```suggestion
            Valid for: paused, running workflows.
```